mrmustard/lab_dev/states/base.py

```diff
@@ -656,8 +656,10 @@ def from_phase_space(
         r"""
         Initializes a density matrix from the covariance matrix, vector of means and a coefficient,
         which parametrize the s-parametrized phase space function
+        ``coeff * exp((x-means)^T cov^-1 (x-means))``.
```

Comment:
Should there be a -1/2 in the exponential? Otherwise, I worry the function isn't bounded. the `:math: `below should also be changed if this is true.

```suggestion
        ``coeff * exp((-1/2)*(x-means)^T cov^-1 (x-means))``.
```